src/components/analyses/details/ParamsPanel.js

```diff
@@ -27,9 +47,72 @@ import {
     Typography,
 } from "@material-ui/core";
 
+function InputParameterValue(props) {
+    const { id, param_type, path } = props;
+
+    const [statError, setStatError] = React.useState();
+
+    React.useEffect(() => {
+        if (!path) {
+            setStatError(null);
+        }
+    }, [path]);
+
+    const displayValue = parseNameFromPath(path);
+    const linkTarget = getFolderPage(
+        AppParamTypes.FOLDER_INPUT === param_type ? path : getParentPath(path)
+    );
```

Comment:
Should we rather open the file in viewer cos the file could be buried down cos pagination although am not sure how useful it is to open a file if it is binary 🤔

Reply:
Right, I'm not sure if users will want to view input contents from here.

I thought the main usefulness of this feature was the tooltip to show where the input was located, or an error message if the input no longer exists (to explain why it would be missing in a relaunch).

Maybe we can stick with the link taking the user to the folder listing for now, and if users start asking to view the contents from here instead, we can change it then. 🤔

Reply:
I got sci-analyst feedback preferring navigating to the parent folder, so I'll stick with that for now.